README.md

```diff
@@ -25,8 +25,8 @@ We'll be at Wagtail Space US this year! The Call for Participation and Registrat
 Wagtail Localize requires the following:
 
 - Python (3.8, 3.9, 3.10, 3.11)
-- Django (3.2, 4.2, 5.0)
-- Wagtail (5.2, 6.0) with [internationalisation enabled](https://docs.wagtail.org/en/stable/advanced_topics/i18n.html#configuration)
+- Django (4.2, 5.0)
+- Wagtail (5.2, 6.0, 6.1) with [internationalisation enabled](https://docs.wagtail.org/en/stable/advanced_topics/i18n.html#configuration)
```

Comment:
Wagtail 6.0 is no longer supported (https://endoflife.date/wagtail) and so it should be removed. This is similar to dropping Django < 4.2

Reply:
Thanks, updated in d98673e